common/jmeter/perf-test-is.sh

```diff
@@ -411,6 +411,26 @@ function print_durations() {
     printf "Script execution time: %s\n" "$(format_time $(measure_time "$test_start_time"))"
 }
 
+function run_b2b_test_data_scripts() {
+
+    echo "Running b2b test data setup scripts"
+    echo "=========================================================================================="
+    declare -a scripts=("TestData_Add_Sub_Orgs.jmx" "TestData_Add_B2B_OAuth_Apps.jmx" "TestData_SCIM2_Add_Sub_Org_Users.jmx")
+    setup_dir="/home/ubuntu/workspace/jmeter/setup"
+
+    for script in "${scripts[@]}"; do
+        script_file="$setup_dir/$script"
+        test_data_store="test-data/$script"
+        mkdir -p $test_data_store
+        command="jmeter -Jhost=$lb_host -Jport=$is_port -JtokenIssuer=$token_issuer -JjwtTokenUserPassword=$jwt_token_user_password -JjwtTokenClientSecret=$jwt_token_client_secret -JnoOfNodes=$noOfNodes -n -t $script_file"
```

Comment:
Do we need JjwtTokenUserPassword and JjwtTokenClientSecret for these setup scripts?
Remove unused values

Reply:
Addressed with cc345a9a29f0f4f1d7ac39add41d91ccc3176847